src/lib/icons.js

```diff
@@ -2,11 +2,15 @@
 // with a possible default icon
 
 export const getKonnectorIcon = konnector => {
+  let icon = null
   if (konnector.icon) {
-    return require(`../assets/konnectors/${konnector.icon}`)
+    try {
+      icon = require(`../assets/konnectors/${konnector.icon}`)
```

Comment:
Here we will use the icon from the stack like for the applications
Here is the usage in the store: https://github.com/cozy/cozy-store/blob/master/src/ducks/apps/index.js#L146
It will be on `/konnectors/:slug/icon`.

Reply:
Well spotted, I suggest to implement it when it will be available from the stack, to keep a Collect a little bit "clean" until then 👍